app/plugin_api.go

```diff
@@ -307,6 +307,10 @@ func (api *PluginAPI) GetPost(postId string) (*model.Post, *model.AppError) {
 	return api.app.GetSinglePost(postId)
 }
 
+func (api *PluginAPI) GetPostsForChannel(channelId string, page, perPage int) (*model.PostList, *model.AppError) {
```

Comment:
@jwilander, thoughts on calling this `GetPostsPageForChannel` to mirror the `app.GetPostsPage` call and allow for a future `GetPostsForChannel` that does `app.GetPosts`, or are we officially moving to the `page, perPage` model and away from `offset, limit`?

Reply:
I'd prefer if we went towards one or the other (1/5 leaning towards page/perPage as it's clearer I think), I created a ticket here to make it all consistent https://mattermost.atlassian.net/browse/MM-12476 which we should do before these new APIs ship in 5.6

Reply:
HI @lieut-data & @jwilander, Do I have to change it to offset/limit?